docs/roman/stpipe/call_via_call.rst

```diff
@@ -1,72 +0,0 @@
-.. _call_examples:
```

Comment:
Would you restore this file?

I think there's some confusion over https://github.com/spacetelescope/romancal/pull/1499 That PR removed uses of `__call__` which is now deprecated in stpipe main. `call` is still the recommended way of using pipelines/steps.